src/components/app/service.ts

```diff
@@ -236,10 +236,19 @@ export function getRollbackMaterialList(cdMaterialId, offset: number, size: numb
     })
 }
 
-function cdMaterialListModal(artifacts: any[], markFirstSelected: boolean) {
+function cdMaterialListModal(artifacts: any[], markFirstSelected: boolean, artifactId?: number, artifactStatus?: string) {
     if (!artifacts || !artifacts.length) return []
+    let artifactStatusValue = ''
 
     let materials = artifacts.map((material, index) => {
+        if(artifactId && artifactStatus){
+            if(material.id === artifactId){
+                artifactStatusValue = artifactStatus
+            } else {
+                artifactStatusValue = ''
```

Comment:
Do we need to assign blank again? Also, don't we need to create local to have status value for all material?

Reply:
done